src/Service.Tests/dab-config.CosmosDb_NoSql.json

```diff
@@ -133,6 +142,63 @@
           ]
         }
       ]
+    },
+    "Earth": {
+      "source": "graphqldb.earth",
+      "permissions": [
+        {
+          "role": "anonymous",
+          "actions": [
+            {
+              "action": "create",
+              "fields": {
+                "include": [
+                  "id"
+                ],
+                "exclude": [
+                  "name"
+                ]
+              }
+            },
+            {
+              "action": "read",
```

Comment:
I see field AuthZ code for Mutation only. Can you please point me where this happens for read ie. Query operations in Cosmos?

Reply:
For read operation, beside Filter and Nested Filter, we relay on the ```@authorized``` directive being added in the ```schema.gql``` which is using hot chocolate query field authorization.